### PR TITLE
Improved the `Epicycloid` class docstring

### PR DESCRIPTION
**Pull request by jbxamora**
## Description
Simple improvement of docstring to make it more technical and concise
Do not include any personal data.

Fixes # (issue)
#86
## Checklist

* [x] I followed the guidelines in our Contributing document
* [x] I added an explanation of my changes

## Additional notes (optional)

### CODE REVIEW

File: spyrograph/epitrochoid/epicycloid.py [hunk 1]
```diff
@@ -9,10 +9,18 @@
...
-    radius of the fixed circle
-    """
+    """A class that represents an epicycloid, which is a curve traced by a point on a circle
+    that rolls around the outside of a fixed circle. This is a special case of an epitrochoid
+    where the rolling circle has a radius that is one-third the radius of the fixed circle.
+    
+    The Epicycloid class allows you to generate points along the curve using mathematical
+    equations, and visualize the curve in a plot. It also provides properties and methods
+    for computing the length, area, and curvature of the curve, and for plotting and animating
+    the curve.
```
Comment (chris-greening) [inline spyrograph/epitrochoid/epicycloid.py:19]: Can you please remove "for computing the length, area, and curvature of the curve, and"

spyrograph currently doesn't do this

File: spyrograph/epitrochoid/epicycloid.py [hunk 2]
```diff
@@ -9,10 +9,18 @@
 from spyrograph._cycloid import _Cycloid
 
 class Epicycloid(_Cycloid, Epitrochoid):
-    """Model of a epicycloid which is a special case of a epitrochoid where the
-    circle is rolling around the outside of the fixed circle and has 1/3 the
-    radius of the fixed circle
-    """
+    """A class that represents an epicycloid, which is a curve traced by a point on a circle
+    that rolls around the outside of a fixed circle. This is a special case of an epitrochoid
+    where the rolling circle has a radius that is one-third the radius of the fixed circle.
```
Comment (chris-greening) [inline spyrograph/epitrochoid/epicycloid.py:14]: Please remove the second sentence as "one-third the radius of the fixed circle." is not relevant to this particular example

**Review by chris-greening — CHANGES_REQUESTED** [review]

**Comment by chris-greening** [comment]
Heyo @jbxamora !! Thanks so much for your contribution, I just flagged some slight tweaks 😄

**Review by jbxamora — COMMENTED** [review]
Got it! Here you go

**Review by jbxamora — COMMENTED** [review]
Pylint

**Review by chris-greening — APPROVED** [review]

**Pull request merged**